migrations/20230504004825_verifiable_credential_tbls.sql

```diff
@@ -0,0 +1,23 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+SET search_path = devices_api, public;
+
+ALTER TABLE vehicle_nfts ADD COLUMN claim_id varchar;
+
+CREATE TABLE verifiable_credentials
+(
+    claim_id varchar primary key,
+    proof bytea not null
```

Comment:
Can you think of another name for this? I feel like the proof is just one section.